utils/dynamodb_handler.py

```diff
@@ -204,23 +204,25 @@ def get_group_count(min_group=1, max_group=30):
     minimal_task_group_count = 200
     for i in range(min_group, max_group + 1):
         ts = resource.Table("Task_group_" + str(i))
-        res = ts.scan(
```

Comment:
basically this wasn't working and always returned Count 0, hence we never got even distribution. Therefore have refactored this logic so that we get evenly distributed responses.